core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -733,12 +733,18 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
       killBroker(0)
       val aliveServers = brokers.filterNot(_.config.brokerId == 0)
       TestUtils.waitForPartitionMetadata(aliveServers, underMinIsrTopic, 0)
-      val output = TestUtils.grabConsoleOutput(
-        topicService.describeTopic(new TopicCommandOptions(Array("--under-min-isr-partitions"))))
+      var output = ""
+      TestUtils.waitUntilTrue(
+        () => {
+          output = TestUtils.grabConsoleOutput(
+            topicService.describeTopic(new TopicCommandOptions(Array("--under-min-isr-partitions"))))
+          2 == output.split("\n").length
+        }, "Couldn't get topics with under-min-isr partitions"
+      )
       val rows = output.split("\n")
+      assertEquals(2, rows.length)
```

Comment:
Why move it upward?

Reply:
So, that if the wait condition times out and we don't get 2 records so instead of getting an ArrayIndexOutOfBounds, we can assert and fail that we didn't receive 2 records. It's minor but I thought it's still better than having a RuntimeException thrown. WDYT?

Reply:
I'm not convinced about this, but both are OK, it doesn't matter.

Reply:
ok.. i moved it back to where it was. It was anyways a cosmetic change..